rpc/handlers.go

```diff
@@ -1178,8 +1171,7 @@ func (h *Handler) TransactionStatus(ctx context.Context, hash felt.Felt) (*Trans
 		case feeder.Reverted:
 			status.Execution = TxnFailure
 		default:
-			// pre-0.12.1
-			status.Execution = TxnSuccess
+			return nil, jsonrpc.Err(jsonrpc.InternalError, "unknown execution status from the feeder gateway")
```

Comment:
Let's not error but omit the field? status.Finality may have some useful information